fraim/cli.py

```diff
@@ -123,6 +140,8 @@ def cli() -> int:
     )
     parser.add_argument("--limit", type=int, help="Limit the number of files to scan")
 
+    parser.add_argument('--show-logs', type=bool, default=False, help='Prints logs to standard error')
```

Comment:
I think we should default to True here for now, or else people that try it out won't see what's happening (and also won't see where the output files are written). Once we implement the cool progress bar type stuff then show-logs defaulting to False could make sense.